platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/LibraryLoader.java

```diff
@@ -5,16 +5,32 @@
 /**
  * Centralises the knowledge about "mapbox-gl" library loading.
  */
-public class LibraryLoader {
+public abstract class LibraryLoader {
+
+  private static final LibraryLoader DEFAULT = new LibraryLoader() {
+    @Override
+    public void load(String name) {
+      try {
+        System.loadLibrary(name);
+      } catch (UnsatisfiedLinkError error) {
+        Timber.e(error, "Failed to load native shared library.");
```

Comment:
Perhaps the catching of the `UnsatisfiedLinkError` + logging should still happen in `LibraryLoader.load()` instead of just the `DEFAULT` implementation.